mlocal/frags/Makefile.stub

```diff
@@ -77,9 +82,8 @@ test:
 	@echo " TEST sudo go test [all]"
 	$(V)M=0; eval 'while [ $$M -le 20 ]; do sleep 60; M=`expr $$M + 1`; echo "Still testing ($$M) ..."; done &' ; \
     	trap "kill $$! || true" 0; \
-		cd $(SOURCEDIR) && \
-		sudo -E \
-		scripts/go-test -v -tags 'integration_test' \
+		cd $(SOURCEDIR) && sudo -v && \
+		scripts/go-test -exec $(test_sudo) -v -tags 'integration_test' \
```

Comment:
I have to _guess_ (hint, hint) that the intention here is to _not_ run the `go` command as root and instead run only the test binary as root. This is pretty good.